docs/Other SDL Features/Getting In-Car Microphone Audio/index.md

```diff
@@ -144,3 +145,114 @@ sdlManager.send(request: performAudioPassThru) { (request, response, error) in
     // Process audio data
 }
 ```
+!@
+
+@![android, javaSE, javaEE]
+`// TODO Android should align closer to iOS, most sections are different or does not include some information that iOS does or the ordering is different`
```

Comment:
Some of this can be fixed by us, like reordering. This should be aligned more closely.